src/services/query-options.js

```diff
@@ -231,8 +230,6 @@ class QueryOptions {
     const primaryKey = _.values(this._model.primaryKeys)[0].field;
     const queryToFilterRecords = query.trim();
 
-    new LiveQueryChecker().perform(queryToFilterRecords);
-
```

Comment:
Pourquoi supprimer la vérification de LiveQuery ?

Reply:
Moved to `forestadmin-client`